apps/vc-api/docs/example-exchange-definitions/presentation-self-signed-exchange.md

```diff
@@ -0,0 +1,91 @@
+Sample Exchange Definition for presentation and self-signed Verifiable Credential. 
+
+Use case: The Holder is required to present a self-signed consent VC and a PermanentResidentCard VC (Issued by Authority) to Verifier.
+
+```json
+{
+   "exchangeId":"286bc1e0-f1bd-488a-a873-8d71be3c690e",
+    "query":[
+      {
+         "type":"PresentationDefinition",
+         "credentialQuery":[
+            {
+               "presentationDefinition":{
+                  "id":"286bc1e0-f1bd-488a-a873-8d71be3c690e",
+                  "submission_requirements":[
+                     {
+                        "name":"Consent and Resident-card Exchange",
+                        "rule":"pick",
+                        "min":2,
+                        "from":"A"
+                     }
+                  ],
+                  "input_descriptors":[
+                     {
+                        "id":"PermanentResidentCard",
+                        "name":"PermanentResidentCard",
+                        "purpose":"PermanentResidentCard",
+                        "group":[
+                           "A"
+                        ],
+                        "constraints":{
+                           "fields":[
+                              {
+                                 "path":[
+                                    "$.type"
+                                 ],
+                                 "filter":{
+                                    "type":"array",
+                                    "contains":{
+                                       "type":"string",
+                                       "const":"PermanentResidentCard"
+                                    }
+                                 }
+                              }
+                           ]
+                        }
+                     },
+                     {
+                        "id":"ConsentCredential",
+                        "name":"ConsentCredential",
+                        "purpose":"One consent credential is required for this presentation",
+                        "group":[
+                           "A"
+                        ],
+                        "constraints":{
+                           "subject_is_issuer":"required",
+                           "fields":[
+                              {
+                                 "path":[
+                                    "$.type"
+                                 ],
+                                 "filter":{
+                                    "type":"array",
+                                    "contains":{
+                                       "type":"string",
+                                       "const":"ConsentCredential"
+                                    }
+                                 }
+                              }
+                           ]
+                        }
+                     }
```

Comment:
Hmm, I used this set of constraints so that it doesn't become too technical for the devs working on the mobile app. Maybe we add the `constraints` for the `ConsentCredential` and add another file to describe what the input descriptor is validating?